mission/config/ui/taskroster/playerInfo/ui.hpp

```diff
@@ -0,0 +1,546 @@
+			text = "Walk up to a duty officer and press the 6 key while looking at them. This will bring up a wheel menu where you can click on the role(s) you want. Duty officers can be located by the white dot markers on the map around the main base.";
+			size = UIH(0.69);
+			tooltip = "";
+			class Attributes
+			{
+				align = "left";
+				valign = "middle";
+				color = "#000000";
+				colorLink = "#D09B43";
+				font = USEDFONT;
+				size = 1;
+				shadow = 0;
+			};
+		};
+
+		class rhs_img_dutyofficer_map: vn_mf_RscPicture
+		{
+			idc = -1;
+			x = UIW(1);
+			y = UIH(16.5);
+			w = UIW(5);
+			h = UIH(5);
+			text = "custom\taskroster\help\img_duty_officer_map.paa";
+			tooltip = "Duty officers are marked by white dots on the map at the main base.";
+		};
+
+		class rhs_img_dutyofficer_map_caption: vn_mf_RscText
+		{
+			idc = -1;
+			x = UIW(1.5);
+			y = UIH(21.5);
+			w = UIW(5);
+			h = UIH(1);
+			sizeEx = TXT_S;
+			colorText[] = {0, 0, 0, 1};
+			text = "Map Markers";
+		};
+
+		class rhs_img_dutyofficer_unit: vn_mf_RscPicture
+		{
+			idc = -1;
+			x = UIW(7);
+			y = UIH(16.5);
+			w = UIW(5);
+			h = UIH(5);
+
+			text = "custom\taskroster\help\img_duty_officer_unit.paa";
+			tooltip = "A Duty Officer.";
+		};
+
+		class rhs_img_dutyofficer_unit_caption: rhs_img_dutyofficer_map_caption
+		{
+			x = UIW(7.5);
+			text = "Duty Officer";
+		};
+
+		class rhs_img_dutyofficer_wheel: vn_mf_RscPicture
+		{
+			idc = -1;
+			x = UIW(13);
+			y = UIH(16.5);
+			w = UIW(5);
+			h = UIH(5);
+
+			text = "custom\taskroster\help\img_duty_officer_wheel.paa";
+			tooltip = "Pressing the 6 key will bring up the wheel menu where you can gain additional roles.";
+		};
+
+		class rhs_img_dutyofficer_wheel_caption: rhs_img_dutyofficer_map_caption
+		{
+			x = UIW(13.5);
+			text = "Wheel Menu";
+		};
+	};
+};
+
+class vn_tr_disp_showPlayerInfo
+{
+	name = "vn_tr_disp_showPlayerInfo";
+	//If already opened -> Recalling it -> Reloading the Dialog (e.g. like updating the view, without "closing" it)
+	onLoad = "[""onLoad"",_this,""vn_tr_disp_showPlayerInfo"",''] call (uinamespace getvariable 'BIS_fnc_initDisplay'); call vn_mf_fnc_tr_playerInfo_show;";
+	onUnload = "[""onUnload"",_this,""vn_tr_disp_showPlayerInfo"",''] call (uinamespace getvariable 'BIS_fnc_initDisplay'); [] spawn vn_mf_fnc_tr_overview_init;";
+	idd = THISISWRONGONPURPOSE;
+	movingEnable = 1;
+	enableSimulation = 1;
+
+	class ControlsBackground
+	{
+		class folderBackground: vn_mf_RscPicture
+		{
+			idc = -1;
+			x = VN_TR_FOLDER_X;
+			y = VN_TR_FOLDER_Y;
+			w = VN_TR_FOLDER_W;
+			h = VN_TR_FOLDER_H;
+			colorText[] = {1,1,1,1};
+			colorBackground[] = {1,1,1,1};
+			text = "\vn\ui_f_vietnam\ui\taskroster\img\tr_folder_background_sheetL.paa";
+		};
+
+		//MUST be in the background, otherwise it could "pop up", when clicking on it, covering up the cordles
+		class rhs: vn_tr_disp_showPlayerInfo_rhs
+		{
+			idc = VN_TR_PLAYERINFO_RHS_IDC;
+		};
+	};
+
+	class Controls
```

Comment:
Could we script all the fields in this so we can collect the data in a scrollable table format?

TODO / later feature improvement